dashboard/db/migrate/20150128022616_create_ops_tables.rb

```diff
@@ -9,7 +9,7 @@ def change
       t.timestamps
     end
 
-    create_table :cohorts do |t|
+    create_table :cohorts do |t|  # rubocop:disable Style/SymbolProc
```

Comment:
Note that this will disable the check for the remainder of the file.  Any reason not to write this as:
```ruby
create_table :cohorts, &:timestamps
```